certs/certs.go

```diff
@@ -0,0 +1,67 @@
+// Package certs handles persisting and loading certs
+package certs
+
+import (
+	"encoding/base64"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"github.com/levenlabs/locksmith/config"
+	"log"
+	"os"
+	"time"
+)
+
+func init() {
+	if config.CertsFile == "" {
+		return
+	}
+	// make sure we can write to the file
+	if _, err := os.Stat(config.CertsFile); err != nil {
+		if !os.IsNotExist(err) {
+			log.Fatalf("Failed to read certs file: %s", err)
+		}
+		//create the file since it doesn't exist
+		f, err := os.Create(config.CertsFile)
+		if err != nil {
+			log.Fatalf("Failed to create certs file: %s", err)
+		}
+		defer f.Close()
+
+		err = f.Chmod(0600)
+		if err != nil {
+			log.Fatalf("Failed to chmod certs file: %s", err)
+		}
+	}
+}
+
+func SaveCert(hostname, remoteAddr, cert string) error {
+	if config.CertsFile == "" {
+		return nil
+	}
+
+	b, _ := pem.Decode([]byte(cert))
+	if b == nil || len(b.Bytes) == 0 {
+		return errors.New("Error reading PEM certificate")
+	}
+
+	b64cert := base64.StdEncoding.EncodeToString(b.Bytes)
+	t := time.Now()
+	d := fmt.Sprintf("%d-%02d-%02dT%02d:%02d:%02d",
```

Comment:
You should use the `Time.Format` method instead, or just stick to the standard format that `Time.String()` outputs, since it'll be easier to parse later